src/status.js

```diff
@@ -0,0 +1,11 @@
+let tasks = [];
+if (localStorage.getItem('tasks')) {
+  tasks = JSON.parse(localStorage.getItem('tasks'));
+}
+
+function completeStatus(index) {
+  tasks[index].completed = true;
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
```

Comment:
- [ ] Good job so far 👏 It looks like the suggestion from the previous reviewer has not been implemented yet, as he said when you try to check the checkbox to mark the task as completed FOR THE FIRST TIME, It shows the error which is shown below 👇 

![image](https://user-images.githubusercontent.com/90222110/236560444-2d78d012-170b-418d-8be0-c5c2a1906bd2.png)

please debug your function in order to collect that. I saw that after I try to remove that error screen, the functionality works so I suggest that you correct that compilation error 😉